drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -253,6 +254,22 @@ def _learner_raw_score(self):
         correct_count, total_count = self._get_item_stats()
         return correct_count / float(total_count)
 
+    @staticmethod
+    def _get_statici18n_js_url():
+        """
+        Returns the Javascript translation file for the currently selected language, if any.
+        """
+        statici18n_js_url = None
+        lang_code = translation.get_language()
+        if lang_code:
+            text_js = 'public/js/translations/{lang_code}/text.js'
+            country_code = lang_code.split('-')[0]
+            for code in (lang_code, country_code):
```

Comment:
This is simply can be done in an if-else statement, no need for a loop.

Reply:
☝️